sites/friday/src/get-started/here/make-changes.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Make changes to your project
+sidebarTitle: Revisions
+weight: 50
+description: How to work on a daily basis with an {{% vendor/name %}} project?
+---
+
+Now that your project is deployed, you can start making changes to it.
+For example, you might want to fix a bug or add a new feature.
+It's this component of Upsun - preview environments - that you'll come to recognize as a key feature.
+
+In your project, the `main` branch (or `master` if you change it during project creation) always represents the production environment.
```

Comment:
"In your project, the project's `default_branch` (eg `main`, `master`, or whatever you entered during project creation)..."